js-core-2/lesson7.md

```diff
@@ -32,59 +32,60 @@ When we are you solving a problem, one of your main decisions as a developer is
 
```

Comment:
I like the exercises. Could we maybe add a bit more theory to this? I.e. introduce some of the methods they can use `hasOwnProperty` etc. or how they can access properties dynamically by string value: `person["name"]`